pandas/core/generic.py

```diff
@@ -4482,7 +4482,7 @@ def add_suffix(self: NDFrameT, suffix: str) -> NDFrameT:
         # expected "NDFrameT")
         # error: Argument 1 to "rename" of "NDFrame" has incompatible type
         # "**Dict[str, partial[str]]"; expected "Union[str, int, None]"
-        return self.rename(**mapper)  # type: ignore[return-value, arg-type]
+        return self.rename(**mapper)
```

Comment:
likewise (`self._rename`)

Reply:
I have pushed the changes, You can review them whenever you are free.

Reply:
this'll fail CI, can you run the `black` formatter on the files you modified?

Reply:
I haven't installed `black` also I don't know much about it.

Reply:
I have installed it and trying to figure out how it works. Is this document good to refer to https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/